### ISSUE TITLE
Output Window on right side of the screen in Atom 

### ISSUE THREAD

**Issue by TLevesque** [wallabyjs/quokka#267]
First, thank you wallaby team for this awesome dev tool! 

I'm using Quokka with Atom (pro version) and I want to setup the toggle of the output window on the right side of my screen (to split vertically instead of horizontally as it is by default), is it possible? I've not seen any option to do it...  If possible, how? 

Thanks for your help!

**Comment by smcenlly** [comment]
This is not currently possible but will be available in our _next release_ of Quokka for Atom.

You will need to update your Atom config to override where to position the output window, specifying the Atom setting, `quokka-output-position` (see example below).

Note: this feature will only available in Quokka pro version. We will update this issue when we release the next version of Quokka for Atom.

```
"*":
  core:
    telemetryConsent: "no"
    uriHandlerRegistration: "always"
  "exception-reporting":
    userId: "567c04ab-974d-45f0-8046-5ccde1f75211"
  "quokka-output-position": "Right"
  welcome:
    showOnStartup: false
```

**Comment by smcenlly** [comment]
This is now available in the latest version of Quokka for Atom, version 1.0.95.

**Comment by TLevesque** [comment]
Thank you Simon, that works perfectly!! Just maybe a little improvement for later if you can, that would be nice if it was possible to resize the width of the vertical output panel... ;)

**Comment by smcenlly** [comment]
Oops. Thanks for letting us know. We'll let you know when this is fixed.

**Comment by smcenlly** [comment]
This is fixed in version 1.0.99 of Quokka for Atom

**Comment by TLevesque** [comment]
Working perfectly! Thank you very much!!
